components/automate-ui/src/app/pages/job-edit/job-edit.component.ts

```diff
@@ -111,29 +111,27 @@ export class JobEditComponent implements OnDestroy {
     }, {
       validator: profileSelectionRequiredValidator
     });
-    const defaultStart = moment();
-    const defaultEnd = moment(defaultStart).add(1, 'days');
+    const defaultStart = moment.utc();
+    const defaultEnd = moment.utc(defaultStart).add(1, 'days');
     const scheduleGroup = this.fb.group({
       name: ['', Validators.required],
       include: false,
       start: this.fb.group({
         datetime: this.fb.group({
-          month: defaultStart.month() + 1,
+          month: defaultStart.month(),
           date: defaultStart.date(),
           year: defaultStart.year(),
-          hour: this.hours24to12(defaultStart.hour()),
-          minute: defaultStart.minute(),
-          meridiem: defaultStart.hour() - 12 > 0 ? 'PM' : 'AM'
+          hour: defaultStart.hour(),
+          minute: defaultStart.minute()
         })
       }),
       end: this.fb.group({
         datetime: this.fb.group({
-          month: defaultEnd.month() + 1,
+          month: defaultEnd.month(),
           date: defaultEnd.date(),
           year: defaultEnd.year(),
-          hour: this.hours24to12(defaultEnd.hour()),
-          minute: defaultEnd.minute(),
-          meridiem: defaultEnd.hour() - 12 > 0 ? 'PM' : 'AM'
+          hour: defaultEnd.hour(),
+          minute: defaultEnd.minute()
         }),
```

Comment:
Same changes that were made in job-add.component are applied to job-edit.component